WordPress/Classes/Models/Blog+Jetpack.m

```diff
@@ -225,6 +225,15 @@ - (NSNumber *)jetpackDotComID
     return [self jetpackBlogID];
 }
 
+- (NSString *)jetpackAuthToken
+{
+    if ([self isWPcom]) {
+        return [self jetpackAuthToken];
```

Comment:
This always wigs me out when I see swizzling :worried: - calling the "wrong" method. :wink: